test/expected.txt

```diff
@@ -297,6 +297,78 @@
 || 
 || 0 persons listed!
 || ===================================================
+|| Enter command: || [Command entered:  add Name with'arc p/99999999 e/name@arc.com a/1 long, #01-valid address, 890765]
+|| New person added: Name with'arc Phone: 99999999 Email: name@arc.com Address: 1 long, #01-valid address, 890765 Tags: 
+|| ===================================================
+|| Enter command: || [Command entered:  list]
+|| 	1. Name with'arc Phone: 99999999 Email: name@arc.com Address: 1 long, #01-valid address, 890765 Tags: 
+|| 
+|| 1 persons listed!
+|| ===================================================
+|| Enter command: || [Command entered:  add John-Doe p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25]
+|| New person added: John-Doe Phone: 98765432 Email: johnd@gmail.com Address: 311, Clementi Ave 2, #02-25 Tags: 
+|| ===================================================
+|| Enter command: || [Command entered:  list]
+|| 	1. Name with'arc Phone: 99999999 Email: name@arc.com Address: 1 long, #01-valid address, 890765 Tags: 
+|| 	2. John-Doe Phone: 98765432 Email: johnd@gmail.com Address: 311, Clementi Ave 2, #02-25 Tags: 
+|| 
+|| 2 persons listed!
+|| ===================================================
+|| Enter command: || [Command entered:  add John-Doe jr. p/98765432 e/johnd@gmail.com a/311, Clementi Ave 2, #02-25]
+|| New person added: John-Doe jr. Phone: 98765432 Email: johnd@gmail.com Address: 311, Clementi Ave 2, #02-25 Tags: 
+|| ===================================================
+|| Enter command: || [Command entered:  list]
+|| 	1. Name with'arc Phone: 99999999 Email: name@arc.com Address: 1 long, #01-valid address, 890765 Tags: 
+|| 	2. John-Doe Phone: 98765432 Email: johnd@gmail.com Address: 311, Clementi Ave 2, #02-25 Tags: 
+|| 	3. John-Doe jr. Phone: 98765432 Email: johnd@gmail.com Address: 311, Clementi Ave 2, #02-25 Tags: 
+|| 
+|| 3 persons listed!
+|| ===================================================
+|| Enter command: || [Command entered:  add Bj�rn Borg p/98765432 e/borg@gmail.com a/311, Clementi Ave 2, #02-25]
+|| New person added: Bj�rn Borg Phone: 98765432 Email: borg@gmail.com Address: 311, Clementi Ave 2, #02-25 Tags: 
+|| ===================================================
```

Comment:
@okkhoy Some characters in the name are corrupted so I think the encoding is not in UTF-8. Can you help check this again?